images/deno/README.md

```diff
@@ -0,0 +1,49 @@
+# deno
+
+Minimal container image for running deno apps
+
+The image specifies a default non-root `deno` user (UID 65532), and a working directory at `/app`, owned by that `deno` user, and accessible to all users.
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/deno:latest
+```
+
+## Usage Example
+
+Navigate to the [`example/`](./example/) directory:
```

Comment:
I like this example, but the problem is we have a monorepo now, and I don't want people to have to check out everything :/

I'm not sure what to suggest. The Dockerfile is small enough to inline here and I guess it could download the ts?

Reply:
Done!